app/javascript/packs/assessments.js

```diff
@@ -1,4 +1,4 @@
-document.querySelectorAll('.need--status_complete').forEach(e => e.style.display = 'none');
+Array.prototype.slice.call(document.querySelectorAll('.need--status_complete')).forEach((e, i) => e => e.style.display = 'none');
```

Comment:
You don't need to wrap this in a slice; [`NodeList` has a `forEach` method](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach).

Reply:
But doesn't work in IE(there NodesList doesn't support forEach)

Reply:
ohhhh! ok